conf/example.conf

```diff
@@ -40,6 +40,21 @@ cache_type = 'memory'
     # default is '/tmp/trickster'
     # cache_path = '/tmp/trickster'
 
+    # Configuration options when using a BoltDb Cache
+    #[cache.boltdb]
+
+    # cache_path defines the directory location under which the Trickster cache will be maintained
+    # default is '/tmp/trickster'
+    #cache_path = '/tmp/trickster'
+
+    # filename defines the name of the BoltDb database file written to cache_path
+    # default is 'bolt.db'
```

Comment:
Defaulting to `trickster.db` might make more sense to name it after the application rather than the database file type?

Reply:
Though if it's in a trickster directory already, who cares...

Reply:
Yes, I think having the file named `trickster.db`, as a relative path rather than absolute, would be better.

Reply:
This documentation needs to be updated because the code does not allow you to change the file name.